Project.toml

```
@@ -20,7 +20,7 @@ WinReg = "1b915085-20d7-51cf-bf83-8f477d6f5128"
 [compat]
 AxisArrays = "≥ 0.0.6"
 CategoricalArrays = "≥ 0.3.0"
-DataFrames = "≥ 0.11.0"
+DataFrames = "0.19"
```

Better accept later versions, which are unlikely to break things. Otherwise, we're going to end up in dependencies incompatibilities very soon.
```suggestion
DataFrames = "≥ 0.19"
```

---

I have heard conflicting recommendations.  Some feel that it is safer not to use the `"≥ ..."` form in case later versions of the dependency are incompatible.  I agree that not allowing future minor or major version changes means that there is some upkeep required in keeping up with dependencies.  Perhaps that is a motivation to "minimize thine dependencies".

---

Yeah, there's no ideal solution for now. Things will be better once we tag 1.0, as then we will avoid breaking changes for a longer period. Until then I prefer using `≥` since using a strict version requirement is really painful in my experience.